src/ApiService/ApiService/onefuzzlib/VmssOperations.cs

```diff
@@ -161,37 +158,24 @@ public class VmssOperations : IVmssOperations {
         }
     }
 
-    public async Async.Task<IDictionary<Guid, string>> ListInstanceIds(Guid name) {
+    public async Async.Task<IDictionary<Guid, string>?> ListInstanceIds(Guid name) {
```

Comment:
NIT: What do you think of this pattern?

```csharp
public async Async.Task<OneFuzzResult<IDictionary<Guid, string>>> ListInstanceId(...) {
    try {
        var results = new Dictionary<Guid, string>();
        ...
        return OneFuzzResult<Dictionary<Guid, string>>.Ok(results);
    } catch (RequestFailedException ex) when (ex.Status == 404) {
        _log.Verbose($"scaleset does not exist {name:Tag:VmssName}");
        return OneFuzzResult<Dictionary<Guid, string>>.Error(ErrorCode.UNABLE_TO_FIND, "Scaleset not found");
    } catch (Exception e) {
        _log.Exception(e);
        return OneFuzzResult<Dictionary<Guid, string>>.Error(ErrorCode.UNABLE_TO_FIND, "failed to list instance ids");
    }
}
```

We get better handling/logging for exceptions and the error/success case is explicit instead of implicitly through null.